pinot-controller/src/main/java/org/apache/pinot/controller/api/events/MetadataEventNotifierFactory.java

```diff
@@ -26,22 +27,21 @@
 public abstract class MetadataEventNotifierFactory {
   public static final Logger LOGGER = LoggerFactory.getLogger(MetadataEventNotifierFactory.class);
   public static final String METADATA_EVENT_CLASS_CONFIG = "factory.class";
-
-  public abstract void init(PinotConfiguration configuration);
+  public abstract void init(PinotConfiguration configuration, PinotHelixResourceManager pinotHelixResourceManager);
```

Comment:
nit: add one empty line before this statement.

Reply:
done